utils/src/field_helpers.rs

```diff
@@ -95,6 +101,20 @@ impl<F: Field> FieldHelpers<F> for F {
             bits
         })
     }
+
+    fn from_big(big: BigUint) -> Result<F> {
+        let mut bytes = big.to_bytes_le();
+
+        // Pad with zeros if necessary because the BigUint to_bytes_le function gives the smallest possible vector of bytes
+        bytes.resize(32, 0);
+
+        F::deserialize(&mut &bytes[..]).map_err(|_| FieldHelpersError::DeserializeBytes)
+    }
+
+    fn to_big(self) -> BigUint {
+        let bytes = self.to_bytes();
+        BigUint::from_bytes_le(&bytes)
+    }
```

Comment:
as I said in the other review, these functions already exists in `PrimeField`, and the `from_big` has a bug (`bytes.resize` will truncate the length if it's > 32)

Reply:
Fair enough... Yeah, in fact the only thing I needed was a pad to 32 when the length was smaller.

Reply:
the new version pads to the smallest larger multiple of 32 bytes